sklearn/bench.py

```diff
@@ -153,6 +153,12 @@ def parse_args(parser, size=None, loop_types=(),
                         help='Seed to pass as random_state')
     parser.add_argument('--dataset-name', type=str, default=None,
                         help='Dataset name')
+    parser.add_argument('--device', type=str, default='None',
```

Comment:
I guess this params should be enabled in higher level scripts that will execute them - but i don't see this in make? 

Another question - are we going to pass single value for bench.py or this should be tuple so bench.py will iterate via them?

Reply:
At this moment I used only runner.py script. Possible that additional changes should be done in make or somewhere else. Thanks.

Devices are provided as a list in config ``"device": ["None", "host", "cpu", "gpu"],``. See example above. Benchmarks are executing with all devices in the list.